seatunnel-connectors-v2/connector-fake/src/main/java/org/apache/seatunnel/connectors/seatunnel/fake/source/FakeRandomData.java

```diff
@@ -0,0 +1,133 @@
+        if (BOOLEAN_TYPE.equals(fieldType)) {
+            return RandomUtils.nextInt(0, 2) == 1;
+        } else if (BYTE_TYPE.equals(fieldType)) {
+            return (byte) RandomUtils.nextInt(0, Byte.MAX_VALUE);
+        } else if (SHORT_TYPE.equals(fieldType)) {
+            return (short) RandomUtils.nextInt(Byte.MAX_VALUE, Short.MAX_VALUE);
+        } else if (INT_TYPE.equals(fieldType)) {
+            return RandomUtils.nextInt(Short.MAX_VALUE, Integer.MAX_VALUE);
+        } else if (LONG_TYPE.equals(fieldType)) {
+            return RandomUtils.nextLong(Integer.MAX_VALUE, Long.MAX_VALUE);
+        } else if (FLOAT_TYPE.equals(fieldType)) {
+            return RandomUtils.nextFloat(Float.MIN_VALUE, Float.MAX_VALUE);
+        } else if (DOUBLE_TYPE.equals(fieldType)) {
+            return RandomUtils.nextDouble(Float.MAX_VALUE, Double.MAX_VALUE);
+        } else if (STRING_TYPE.equals(fieldType)) {
+            return RandomStringUtils.randomAlphabetic(10);
+        } else if (LocalTimeType.LOCAL_DATE_TYPE.equals(fieldType)) {
+            return randomLocalDateTime().toLocalDate();
+        } else if (LocalTimeType.LOCAL_TIME_TYPE.equals(fieldType)) {
+            return randomLocalDateTime().toLocalTime();
+        } else if (LocalTimeType.LOCAL_DATE_TIME_TYPE.equals(fieldType)) {
+            return randomLocalDateTime();
+        } else if (fieldType instanceof DecimalType) {
+            DecimalType decimalType = (DecimalType) fieldType;
+            return new BigDecimal(RandomStringUtils.randomNumeric(decimalType.getPrecision() - decimalType.getScale()) + "." +
+                RandomStringUtils.randomNumeric(decimalType.getPrecision() - decimalType.getScale()));
+        } else if (fieldType instanceof ArrayType) {
+            ArrayType<?, ?> arrayType = (ArrayType<?, ?>) fieldType;
+            BasicType<?> elementType = arrayType.getElementType();
+            Object value = randomColumnValue(elementType);
+            Object arr = Array.newInstance(elementType.getTypeClass(), 1);
+            Array.set(arr, 0, value);
+            return arr;
+        } else if (fieldType instanceof MapType) {
+            MapType<?, ?> mapType = (MapType<?, ?>) fieldType;
+            SeaTunnelDataType<?> keyType = mapType.getKeyType();
+            Object key = randomColumnValue(keyType);
+            SeaTunnelDataType<?> valueType = mapType.getValueType();
+            Object value = randomColumnValue(valueType);
+            HashMap<Object, Object> objectObjectHashMap = new HashMap<>();
+            objectObjectHashMap.put(key, value);
+            return objectObjectHashMap;
+        } else if (fieldType instanceof PrimitiveByteArrayType) {
+            return RandomUtils.nextBytes(10);
+        } else if (VOID_TYPE.equals(fieldType) || fieldType == null) {
+            return Void.TYPE;
+        } else {
+            throw new IllegalStateException("Unexpected value: " + fieldType);
```

Comment:
Change to UnsupportedOperationException？

Reply:
> Change to UnsupportedOperationException？
done